v2.1/how-to/migrate/overview.md

```diff
@@ -11,11 +11,11 @@ category: how-to
 
 在上述数据迁移过程中会用到如下工具：
 
-- [Mydumper](/tools/mydumper.md)：用于从 MySQL 导出数据。建议使用 Mydumper，而非 mysqldump。
-- [Loader](/tools/loader.md)：用于将 Mydumper 导出格式的数据导入到 TiDB。
-- [Syncer](/tools/syncer.md)：用于将数据从 MySQL 增量同步到 TiDB。
-- [DM (Data Migration)](/tools/dm/overview.md)：集成了 Mydumper、Loader、Syncer 的功能，支持 MySQL 数据的全量导出和到 TiDB 的全量导入，还支持 MySQL binlog 数据到 TiDB 的增量同步。
-- [TiDB-Lightning](/tools/lightning/overview-architecture.md)：用于将全量数据高速导入到 TiDB 集群。例如，如果要导入超过 1TiB 的数据，使用 Loader 往往需花费几十个小时，而使用 TiDB-Lighting 的导入速度至少是 Loader 的三倍。
+- [Mydumper](/reference/tools/mydumper.md)：用于从 MySQL 导出数据。建议使用 Mydumper，而非 mysqldump。
+- [Loader](/reference/tools/loader.md)：用于将 Mydumper 导出格式的数据导入到 TiDB。
+- [Syncer](/reference/tools/syncer.md)：用于将数据从 MySQL 增量同步到 TiDB。
+- [DM (Data Migration)](/reference/tools/dm/overview.md)：集成了 Mydumper、Loader、Syncer 的功能，支持 MySQL 数据的全量导出和到 TiDB 的全量导入，还支持 MySQL binlog 数据到 TiDB 的增量同步。
```

Comment:
It seems this relative path is not correct, please check again

Reply:
Yes. Thanks! Updated in https://github.com/pingcap/docs-cn/pull/1564.